File: 06/title_and_url.py
# -*- coding: utf-8 -*- 
from selenium import webdriver
from time import sleep
import os

dr = webdriver.Firefox()
url = 'http://www.baidu.com'
dr.get(url)

print "title of current page is %s" %(dr.title)
print "url of current page is %s" %(dr.current_url)

sleep(10)

dr.quit()

File: 06/TitleAndUrl.java
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;


public class TitleAndUrl {

	public static void main(String[] args) throws InterruptedException {
		WebDriver dr = new FirefoxDriver();
		Thread.sleep(2000);
		
		String url = "http://www.baidu.com";
		
		dr.get(url);
		Thread.sleep(2000);
		
		System.out.println("title of current page is " +dr.getTitle());
		System.out.println("url of current page is "+dr.getCurrentUrl());
		
		System.out.println("browser will be close");
		dr.quit();	
	}

}
